xdcs-frontend/src/app/view/nodes/nodes.component.ts

```diff
@@ -1,10 +1,30 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { NavbarItem } from '../../services/navbar.service';
+import { NodesService } from '../../../api/services/nodes.service';
+import { NodeDto } from '../../../api/models/node-dto';
+import { first } from 'rxjs/operators';
+import { NodesDto } from '../../../api/models/nodes-dto';
 
 @Component({
-  selector: 'app-node',
+  selector: 'app-nodes',
   templateUrl: './nodes.component.html',
   styleUrls: ['./nodes.component.less'],
 })
-export class NodesComponent {
-}
+@NavbarItem('Nodes')
+export class NodesComponent implements OnInit {
+
+  cards: Array<NodeDto> = [];
```

Comment:
możesz to ustawić na nulla i zamiast sprawdzać queried to sprawdzać czy to nie jest nullem